starky/src/cross_table_lookup.rs

```diff
@@ -440,24 +440,22 @@ impl<'a, F: RichField + Extendable<D>, const D: usize>
     CtlCheckVars<'a, F, F::Extension, F::Extension, D>
 {
     /// Extracts the `CtlCheckVars` from a single proof.
-    pub fn from_proof<C: GenericConfig<D, F = F>>(
+    pub fn from_opening_set<C: GenericConfig<D, F = F>>(
```

Comment:
Method description outdated now.

Also I am not sure I understand the need for changing the API here, if we're using the proof opening set anyway I'd rather keep it unchanged to reduce the friction